components/Tabs/src/MDCTabBar.h

```diff
@@ -84,6 +85,33 @@ IB_DESIGNABLE
 /** Ink color for taps on tab bar items. Default: Semi-transparent white. */
 @property(nonatomic, nonnull) UIColor *inkColor UI_APPEARANCE_SELECTOR;
 
+/**
+ Text attributes used for selected item titles.
+ Default uses the regular font from the font loader.
+ */
+@property(nonatomic, strong, nonnull)
+    NSDictionary<NSAttributedStringKey, id> *selectedItemTitleTextAttributes UI_APPEARANCE_SELECTOR;
```

Comment:
Does it make sense to match UIKit:
- (NSDictionary<NSAttributedStringKey, id>)itemTitleTextAttributesForState;
- (void)setItemTitleTextAttributes:NSDictionary<NSAttributedStringKey, id> forState:UIControlState;

Reply:
Switched back to UIFont